Mathlib/Combinatorics/SimpleGraph/Regularity/Uniform.lean

```diff
@@ -21,15 +21,15 @@ The definition is pretty technical, but it amounts to the edges between `s` and
 The literature contains several definitions which are equivalent up to scaling `ε` by some constant
 when the partition is equitable.
 
-A partition `P` of the vertices is `ε`-uniform if the proportion of non `ε`-uniform pairs of parts
+A partition `P` of the vertices is `ε`-uniform if the proportion of non-`ε`-uniform pairs of parts
```

Comment:
I find this doubly hyphenated adjective hard to read. Is there a rephrasing which is more readable?

Reply:
Looks like a good direction!